Library/Homebrew/cask/lib/hbc/dsl.rb

```diff
@@ -98,6 +100,40 @@ def homepage(homepage = nil)
       @homepage ||= homepage
     end
 
+    def language(*args, default: false, &block)
+      if !args.empty? && block_given?
+        @language_blocks ||= {}
+        @language_blocks[args] = block
+
+        return unless default
+
+        unless @language_blocks.default.nil?
+          raise CaskInvalidError.new(token, "Only one default language may be defined")
+        end
+
+        @language_blocks.default = block
+      else
+        language_eval
```

Comment:
I'll leave this as is because of https://github.com/Homebrew/brew/pull/906/files/3def3b805b139960f11af68e07e3295c6dbe92cb#diff-f85d4948a976b9727959680dbfe1118aR16.

Otherwise I'd have to call `@dsl.language` to actually “evaluate” the languages, which kind of defeats the purpose of having a `language_eval` method.